src/gatsby-node.js

```diff
@@ -1,42 +1,47 @@
-const path = require('path');
-const { exists, writeFile, ensureDir } = require('fs-extra');
+const path = require('path')
+const fs = require('fs')
+const { promisify } = require('util')
 
-const getMetaRedirect = require('./getMetaRedirect');
+const stat = promisify(fs.stat)
+const writeFile = promisify(fs.writeFile)
```

Comment:
> All methods in fs are attached to fs-extra. All fs methods return promises if the callback isn't passed.

We can just use fs-extra for all fs functions instead of promisifying them